components/TCardActions.vue

```diff
@@ -3,26 +3,35 @@
     <TButton
       :type="type"
       icon="fire"
-      label="Report"
+      :label="$t('report')"
       @click="isReportShown = true"
     />
-    <TPopup v-if="isReportShown" title="Report" @close="isReportShown = false">
+    <TPopup
+      v-if="isReportShown"
+      :title="$t('report')"
+      @close="isReportShown = false"
+    >
       <div class="p-4">
         <TField
           v-model="reportCategory"
-          label="Reason"
+          :label="$t('TCardActions.Reason')"
           component="TInputSelect"
-          :options="['spam', 'other']"
+          :options="[
+            $t('TCardActions.options.spam'),
```

Comment:
`TInputSelect` works either with array of values or array of objects `{ label, value }`.
Translated string can't be used as value. You need to change items to look like this:

```suggestion
            { value:'spam', label: $t('TCardActions.options.spam') },
```